packages/opal-server/opal_server/policy/webhook/api.py

```diff
@@ -27,22 +31,22 @@ async def trigger_webhook(
         request: Request,
         urls: List[str] = Depends(func_dependency)
     ):
-        event = request.headers.get('X-GitHub-Event', 'ping')
-
-        if (opal_server_config.POLICY_REPO_URL is not None and opal_server_config.POLICY_REPO_URL in urls)\
-                or \
-                (source_type == PolicySourceTypes.Api):
-            if source_type == PolicySourceTypes.Api:
-                logger.info("Triggered webhook to check API bundle URL")
-                response = {"status": "ok", "remote": opal_server_config.POLICY_BUNDLE_URL}
-            elif source_type == PolicySourceTypes.Git:
-                logger.info("triggered webhook on repo: {repo}", repo=urls[0], hook_event=event)
-                response = {"status": "ok", "event": event, "repo_url": urls[0]}
-            if event == 'push' or source_type == PolicySourceTypes.Api:
-                # notifies the webhook listener via the pubsub broadcaster
-                await pubsub_endpoint.publish(opal_server_config.POLICY_REPO_WEBHOOK_TOPIC)
-            return response
+        # TODO: breaking change: change "repo_url" to "remote_url" in next major
+        if source_type == PolicySourceTypes.Git:
+            event = request.headers.get('X-GitHub-Event', 'ping')
+
+            if opal_server_config.POLICY_REPO_URL is not None and opal_server_config.POLICY_REPO_URL in urls:
+                logger.info("triggered webhook on repo: {repo}", repo=opal_server_config.POLICY_REPO_URL, hook_event=event)
+                if event == 'push':
+                    # notifies the webhook listener via the pubsub broadcaster
+                    await pubsub_endpoint.publish(opal_server_config.POLICY_REPO_WEBHOOK_TOPIC)
+                return { "status": "ok", "event": event, "repo_url": opal_server_config.POLICY_REPO_URL }
+
+        elif source_type == PolicySourceTypes.Api:
+            logger.info("Triggered webhook to check API bundle URL")
+            await pubsub_endpoint.publish(opal_server_config.POLICY_REPO_WEBHOOK_TOPIC)
+            return {"status": "ok", "event": "webhook_trigger", "repo_url": opal_server_config.POLICY_BUNDLE_URL}
```

Comment:
this changes are mainly to make the code more readable - i.e: separate the cases where the policy source is git or an api bundle server.